File: UI/trackit_mobile/lib/models/ActivityLevel/activity_level.dart
import 'package:json_annotation/json_annotation.dart';
part 'activity_level.g.dart';

@JsonSerializable()
class ActivityLevel {
  int? activityLevelId;
  String? name;
  double? multiplier;
  String? image;

  ActivityLevel(this.activityLevelId, this.name, this.multiplier, this.image);

  factory ActivityLevel.fromJson(Map<String, dynamic> json) =>
      _$ActivityLevelFromJson(json);

  Map<String, dynamic> toJson() => _$ActivityLevelToJson(this);
}

File: TrackIt.Model/Models/ActivityLevel.cs
﻿namespace TrackIt.Model.Models
{
	public class ActivityLevel
	{
		public int ActivityLevelId { get; set; }

		public string Name { get; set; } = null!;

		public double Multiplier { get; set; }

		public byte[]? Image { get; set; }
	}
}
